schema/Mutation.graphql

```diff
@@ -84,4 +84,15 @@ type Mutation {
     Revoke user session
     """
     revokeUserSession(id: ObjectID!, displayNotice: Boolean): Boolean!
+
+    """
+    Create a new Life object
+    """
+    createlife(
+        firstName: String!, 
+        lastName: String!,
+        description: String!,
+        title: String!
+        birthday: DateTime!,
+        hobbies: [String!]!): Life!
```

Comment:
You can create an `Input` object here instead of having multiple arguments
https://graphql.org/graphql-js/mutations-and-input-types/